Assets/Plugins/Source/Core/PlatformSpecifics.cs

```diff
@@ -21,15 +21,21 @@
  */
 
 #if !EOS_DISABLE
+#if !UNITY_EDITOR
+using UnityEngine.Scripting;
+[assembly: AlwaysLinkAssembly]
+#endif
 namespace PlayEveryWare.EpicOnlineServices
 {
     using System;
     using System.Collections.Generic;
     using System.IO;
     using UnityEngine;
+    using UnityEngine.Scripting;
```

Comment:
Is the twice-included UnityEngine.Scripting necessary?

Reply:
It's not - but I'm not seeing the duplicate, is it out of scope from the preview here?